plugins/integrations/kubernetes-service/src/main/java/com/cloud/kubernetes/version/KubernetesVersionManagerImpl.java

```diff
@@ -202,6 +204,10 @@ public static int compareSemanticVersions(String v1, String v2) throws IllegalAr
         return 0;
     }
 
+    public static boolean versionSupportsAutoscaling(KubernetesSupportedVersion clusterVersion) {
```

Comment:
For UI, do we need to return this in KubernetesSupportedVersionResponse? Or can we deprecate older k8s version?

Reply:
@shwstppr Do you mean as a flag ?

Reply:
@davidjumani something like this https://github.com/shapeblue/cloudstack/blob/41a7eac5f55acfe85f7f32443f1b8ef8574392f5/plugins/integrations/kubernetes-service/src/main/java/org/apache/cloudstack/api/response/KubernetesSupportedVersionResponse.java#L63?

Reply:
Makes sense to deprecate it. Can also add a flag in case users are running older versions. Thanks @shwstppr